File: app/src/main/java/br/com/gabryel/financeapp/investment/Wallet.java
package br.com.gabryel.financeapp.investment;

/**
 * Representation of an Wallet
 * <p/>
 * Created by gabryel on 04/04/16.
 */
public interface Wallet extends Moneyable {

	/**
	 * Add a investment to this wallet
	 *
	 * @param investment Internal investment data
	 */
	void add(Moneyable investment);
}

File: app/src/main/java/br/com/gabryel/financeapp/investment/data/Wallet.kt
package br.com.gabryel.financeapp.investment.data

/**
 * Representation of an Wallet
 *
 * Created by gabryel on 04/04/16.
 */
interface Wallet : Moneyable {

    /**
     * Add a investment to this wallet
     *
     * @param investment Internal investment data
     */
    fun add(investment: Moneyable)
}
